my-app/src/containers/CounterContainer.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types'
+import Counter from '../views/counter/index';
+
+class CounterContainer extends React.Component {
+
+    constructor() {
+        super()
+        this.state = {
+            count: 0
+        }
+    }
+
+    incrementCounter = () => {
+
+        this.setState({
+            count: this.state.count + 1
```

Comment:
Save this.state.count into a separate variable because using state in this way can cause hidden side effects.